keyboards/nyquist/keymaps/chaitanya/keymap.c

```diff
@@ -0,0 +1,197 @@
+#include "nyquist.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```